### PR TITLE
Fixed a bug in the MAB scheme code when skip is None

### PR DESCRIPTION
**Pull request by SHZ66**
**Issue Number.** Is this pull request related to any outstanding issues? If so, list the issue number.  

**Describe the changes made.** A clear and concise description of what the problem is and what you did to fix it. E.g. [...] was happening and I've changed [...] to fix it.  

MAB scheme can crash when `skip` is unspecified in the config file. This PR fixes it by creating a list of `False`'s when `skip` is `None`.

**Major files changed.**  
- [x] src/westpa/core/binning/mab.py

**Status.**
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the CONTRIBUTING document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

**Additional context.** Add any other context or screenshots about the pull request here.

### CODE REVIEW

File: src/westpa/core/binning/mab.py [hunk 1]
```diff
@@ -24,6 +24,9 @@ def map_mab(coords, mask, output, *args, **kwargs):
     if not np.any(mask):
         return output
 
+    if skip is None:
+        skip = [False] * ndim
```
Comment (jeremyleung521) [inline src/westpa/core/binning/mab.py]: Since we're using 0 and 1s for skip, this should be `skip = [0] * ndim` for consistency

**Review by jeremyleung521 — CHANGES_REQUESTED** [review]
Just a code consistency thing.

File: src/westpa/core/binning/mab.py (hunk 1 above)
Comment (SHZ66) [reply src/westpa/core/binning/mab.py]: Nice catch. Just fixed it.

**Review by jeremyleung521 — APPROVED** [review]

**Comment by SHZ66** [comment]
@jeremyleung521 Thanks for reviewing and approving the PR!

**Pull request merged**